### ISSUE TITLE
Add cloud sync

### ISSUE THREAD

**Issue by gcampax** [stanford-oval/almond-gnome#19]
Cloud sync is nice as a way to backup all devices (in case e.g. the db gets corrupted on upgrades), and also nice to have the same configuration on multiple accounts.

**Comment by jeukel** [comment]
I don't see the point on having and app on the phone an one on the desktop that are completely unrelated. As a local server or using the Standford service therefore, will be waiting for the cloud sync option. :-). Thanks.

**Comment by gcampax** [comment]
Hi Daniel!
Indeed that's true! Most of the infrastructure is there in fact - that's why you can sync & backup the phone app to the cloud. It's a matter of wiring up the UI before we can enable this feature here too.
In fact, this issue is primarily a "TODO reminder" that we should do that - but also a suggestion for new contributors ;)
